### PR TITLE
Send logs from Django, Celery & NodeJS containers to Cloudwatch.

### PR DESCRIPTION
**Pull request by KhalidRmb**
@deshraj @Ram81 @RishabhJain2018 
https://docs.docker.com/config/containers/logging/awslogs/

### CODE REVIEW

File: docker-compose-production.yml [hunk 1]
```diff
@@ -9,6 +9,12 @@ services:
       dockerfile: docker/prod/django/Dockerfile
     ports:
       - "8000:8000"
+    logging:
+      driver: awslogs
+      options:
+        awslogs-region: ${AWS_DEFAULT_REGION}
+        awslogs-group: django_production
```
Comment (Ram81) [inline docker-compose-production.yml:16]: rest of the `awslogs-group` field are in format `"<log_group>"`. This one isn't, can we keep it consistent for all fields.

File: docker-compose-production.yml [hunk 2]
```diff
@@ -9,6 +9,11 @@ services:
       dockerfile: docker/prod/django/Dockerfile
     ports:
       - "8000:8000"
+    logging:
+      driver: awslogs
+      options:
+        awslogs-region: ${AWS_DEFAULT_REGION}
+        awslogs-group: "django_production"
```
Comment (RishabhJain2018) [inline docker-compose-production.yml:16]: I think we also need to add this option here: https://docs.docker.com/config/containers/logging/awslogs/#awslogs-create-group

Same for all below options

Comment (KhalidRmb) [reply docker-compose-production.yml:16]: I've created those log groups on AWS, since it's not dynamic. We don't need the option here.

**Comment by KhalidRmb** [comment]
@RishabhJain2018 Let's get this merged?

File: docker-compose-production.yml (hunk 2 above)
Comment (RishabhJain2018) [reply docker-compose-production.yml:16]: But what if someone forks EvalAI and tries to deploy in the production setting or we change our AWS account then we will have to create them manually. What is the issue in adding this option?

Comment (KhalidRmb) [reply docker-compose-production.yml:16]: Right, I didn't realize that. I'll add the option.

Comment (RishabhJain2018) [reply docker-compose-production.yml:16]: Thanks!

**Pull request merged**